Formula/editorconfig-checker.rb

```diff
@@ -0,0 +1,19 @@
+class EditorconfigChecker < Formula
+  desc "Tool to verify that your files are in harmony with your .editorconfig"
+  homepage "https://github.com/editorconfig-checker/editorconfig-checker"
+  url "https://github.com/editorconfig-checker/editorconfig-checker/archive/refs/tags/#{version}.tar.gz"
+  version "2.5.0"
+  sha256 "50274a09c760303bc88c0ead319eda4bc6f317764a1cade1ea86e9ff17d7c0b1"
+  license "MIT"
+  head "https://github.com/editorconfig-checker/editorconfig-checker.git", branch: "main"
+
+  depends_on "go@1.16" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-X main.version=#{version}", output: bin/"editorconfig-checker"), "./cmd/editorconfig-checker/main.go"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/editorconfig-checker --version")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
thanks, I added a simple test to check a file.